client/components/repository/common/ActivityDiscussion.vue

```diff
@@ -37,7 +33,10 @@ export default {
       return orderBy(comments, 'createdAt', 'desc');
     },
     unseenComments: vm => vm.getUnseenActivityComments(vm.activity),
-    lastCommentAt: vm => new Date(get(vm.comments[0], 'createdAt', 0)).getTime()
+    lastCommentAt: vm => new Date(get(vm.comments[0], 'createdAt', 0)).getTime(),
+    sheetOptions: vm => vm.panel
```

Comment:
Consider renaming this computed prop to something that would indicate that these are _style_ options.